albumentations/augmentations/geometric/functional.py

```diff
@@ -310,3 +311,52 @@ def longest_max_size(img, max_size, interpolation):
 @preserve_channel_dim
 def smallest_max_size(img, max_size, interpolation):
     return _func_max_size(img, max_size, interpolation, min)
+
+
+@preserve_channel_dim
+def perspective(img, matrix, max_width, max_height, border_val, border_mode, keep_size, interpolation):
```

Comment:
Let's add types to the functions?

Reply:
Added

Reply:
I don't have much experience with type hints, so if you check it out I will be grateful.

Reply:
Something like this would be fine to explain to newscomers what's expected as input
```
def perspective(img: np.ndarray, matrix: np.ndarray, max_width:int, max_height:int, border_val:int, border_mode:int, keep_size:bool, interpolation:int) -> np.ndarray:
```

Reply:
But `border_val` maybe be any type from this list: `Union[int, float, List[int], List[float], np.ndarray]`
You think that better to simplify construction and set only one type  - `int`?

Reply:
So you telling me you don't have much experience with type hints? ;) ^_^
Well, yes, for border_val we can use all that. OR. Even better solution - we can define alias BorderTypes = Union[int, float, List[int], List[float], np.ndarray] and use it everywhere for this type of argument.

Reply:
I just read the manual 😄 
I didn't know about types aliases.
I think is better to use the current solution in this PR. And when we refactor other transforms and functions we can add an alias.